app/models/client_search_util/name_search.rb

```diff
@@ -0,0 +1,152 @@
+module ClientSearchUtil
+  class NameSearch
```

Comment:
```suggestion
  class NameSearch
    include ArelHelper
```

Reply:
Actually, never mind.  I was going to suggest we cleanup the SQL that's easy to cleanup into Arel, but I think everything here would be somewhat tricky to write.